homeassistant/components/huawei_lte/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "huawei_lte",
-  "name": "Huawei LTE",
+  "name": "Huawei/SoyeaLink LTE",
```

Comment:
We have a concept in HA for supporting multiple brands from a single integration: `supported_brands`, I think we should use that instead.